Druid/DRLoadVectorTemporaryVariable.class.st

```diff
@@ -0,0 +1,13 @@
+Class {
+	#name : 'DRLoadVectorTemporaryVariable',
+	#superclass : 'DRLoadTemporaryVariable',
+	#category : 'Druid-IR',
+	#package : 'Druid',
+	#tag : 'IR'
+}
+
+{ #category : 'testing' }
+DRLoadVectorTemporaryVariable >> isTempVectorTemp [
+	
+	^ true
+]
```

Comment:
Is it justified a subclass just for a boolean? Anyway I think that the `DRTemporaryVariable` will have a `scope` soon, and this class should not be necessary.